src/controllers/Flashcards.js

```diff
@@ -0,0 +1,99 @@
+const Flashcard = require('../models/Flashcard');
+const { StatusCodes } = require('http-status-codes');
+
+const getAllFlashcards = async (req, res) => {
+    const flashcards = await Flashcard.find({}).sort('createdAt');
+    res.status(StatusCodes.OK).json({ flashcards, count: flashcards.length });
+};
+
+const getUserFlashcards = async (req, res) => {
+    const flashcards = await Flashcard.find({ createdBy: req.user.userId }).sort('createdAt');
+    res.status(StatusCodes.OK).json({ flashcards, count: flashcards.length });
+};
+
+const getFlashcard = async (req, res) => {
+    const { 
+        user: { userId }, 
+        params: { id: flashcardId } 
+    } = req;
+
+    console.log("User ID:", userId);
```

Comment:
[Debug Code]
Looks like you left debug code in here.

Reply:
replaced debug code with 400 error